.github/workflows/conformance-gateway-api.yaml

```diff
@@ -262,7 +266,7 @@ jobs:
               -v ./operator/pkg/gateway-api \
               --gateway-class cilium \
               --all-features \
-              --exempt-features "${{ steps.vars.outputs.exempt-features }}" \
```

Comment:
Do we still need `exempt-features` in the workflow?

Reply:
yes, it's still used for normal run (but not profile based run)

Reply:
Oh, I see, thanks!